course/layouts/partials/course_content.html

```diff
@@ -28,7 +28,7 @@ <h1 class="text-uppercase font-weight-bold pb-1 mb-3">{{ .Title }}</h1>
     </div>
   </header>
   {{ partial "mobile_nav_toggle.html" . }}
-  <article class="content pt-3 mt-1">
+  <article class="content pt-3 mt-1 mr-5">
```

Comment:
I think we should give `right margin` on mobile screens only as mentioned in the issue ticket. 
Maybe we can give equal margins on both sides (left and right) (if it looks good) so there is no inconsistency in left/right margins on mobile screens

Reply:
Since this change is in a partial, does it mean that courses will need to be republished to pick up the change?

Reply:
No, since this is a change in ocw-hugo-themes styling